metadata_service/__main__.py

```diff
@@ -18,13 +18,15 @@
 from typing import Optional
 import typer
 import uvicorn
+from ghga_service_chassis_lib.api import run_server
+from metadata_service.config import get_config
 from metadata_service.api import app
 
 
 def run(config: Optional[str] = typer.Option(None, help="Path to config yaml.")):
     """Starts backend server"""
-    # TODO: implement config
-    uvicorn.run(app)
+    #uvicorn.run(app)
```

Comment:
As far as I remember, we agreed on not keeping legacy code in comments

Reply:
Thanks @MoritzHahn1337 ! Legacy code removed.